VndbSharp/Enums/VndbFlags.cs

```diff
@@ -21,13 +21,15 @@ public enum VndbFlags
 		Stats = 1 << 5,
 		[Description("screens")]
 		Screens = 1 << 6,
-		[Description("vns")]
-		VisualNovels = 1 << 7,
+		[Description("vn")]
+		VisualNovel = 1 << 7,
```

Comment:
I was just putting the flags in the order that they appeared on the vndb's API page. Really, nothing more than that.